.golangci.yaml

```diff
@@ -42,6 +42,7 @@ linters:
     - bodyclose
     - containedctx
     - contextcheck
+    - copyloopvar
```

Comment:
`copyloopvar` and `intrange` are enabled, but just give a warning that they don't do anything unless using at least Go 1.22 (we're still using Go 1.17). We definitely don't want to upgrade beyond the features we need, since that would needlessly require users to upgrade.